cpp/src/io/csv/csv_reader.cu

```diff
@@ -61,13 +60,13 @@
 #include "NVStrings.h"
 
 constexpr int32_t HASH_SEED = 33;
+constexpr size_t max_chunk_bytes = 64*1024*1024; // 64MB
 
 using namespace std;
```

Comment:
Don't use `using namespace` at global scope. It almost always eventually leads to problems.